.github/workflows/ci.yml

```diff
@@ -69,7 +69,8 @@ jobs:
           path: nuget/
 
   publish:
-    if: ((github.event_name == 'workflow_dispatch') && (github.ref == 'refs/heads/master') && endsWith(github.actor, '-stripe'))
+    name: Publish
+    if: (((github.event_name == 'push') || (github.event_name == 'workflow_dispatch')) && startsWith(github.ref, 'refs/tags/v') && endsWith(github.actor, '-stripe'))
```

Comment:
Double checking here that we're OK with having workflow_dispatch only work on ref/tags/ rather than master

Reply:
Yep, now that the workflow is going to be present in tag commit we should be good.